modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -1447,4 +1448,20 @@ private static class Augmentation {
             this.addition = addition;
         }
     }
+
+    private static Map<String, Integer> extractDataNodes(Entry dataNodesEntry) {
+        if (!dataNodesEntry.empty()) {
```

Comment:
So, dataNodesEntry won't be null, is that correct?

Reply:
Yes.